recipes/LibriSpeech/ASR/transformer/train_with_whisper.py

```diff
@@ -0,0 +1,357 @@
+#!/usr/bin/env python3
+""" Take the encoder-decoder Whisper and the official tokenizer of whisper to fine-tune it with the NLL.
+
+This recipe can also be used as a zero-shot recipe by removing the training part.
```

Comment:
More ref / text on the header (see the CTC whisper header)

Reply:
Done.